docs/src/components/productX/XHero.tsx

```diff
@@ -117,8 +117,8 @@ export default function XHero() {
             components. We&apos;re kicking it off with the most powerful Data Grid on the market.
           </Typography>
           <GetStartedButtons
-            installation="npm install @mui/x-data-grid"
-            to={ROUTES.dataGridDocs}
+            installation=""
```

Comment:
I worked on a similar change for https://mui.com/ 👌: #37971. I propose we use the same implementation of `GetStartedButtons.tsx`